### PR TITLE
Fixes for sphinx 1.7.x

### PR DESCRIPTION
**Pull request by luisbarrancos**
This should address https://github.com/spinus/sphinxcontrib-images/issues/47 and https://github.com/spinus/sphinxcontrib-images/issues/41

### CODE REVIEW

**Comment by spinus** [comment]
@luisbarrancos thank you. I tried to resolve conflicts but github still find some stuff and ask for manual intervention. Would you mind rebasing this to master?
Please add also sphinx 1.7.9 to test stack (both .travis and tox.ini)

**Comment by luisbarrancos** [comment]
@spinus done

File: .travis.yml [hunk 1]
```diff
@@ -32,6 +34,14 @@ matrix:
       env: TOXENV=py34-sphinx13b2
     - python: 3.4
       env: TOXENV=py34-sphinx133
+    - python: 3.4
+      env: TOXENV=py34-sphinx113
```
Comment (spinus) [inline .travis.yml]: why this duplication :-)?

**Comment by spinus** [comment]
there is one small question and after that I'm ready to merge and do new release

**Comment by luisbarrancos** [comment]
> there is one small question and after that I'm ready to merge and do new release

fixed (overenthusiastic vim fetichism).

**Comment by t-b** [comment]
@luisbarrancos Looks like travis does not like python 3.7/3.8?!

**Comment by spinus** [comment]
right, but that seem to be some internal issue, def not a blocker

**Comment by bariod** [comment]
what about a release then? :thinking: @spinus

**Comment by spinus** [comment]
@bariod Done!

**Comment by spinus** [comment]
thanks for reminder!

**Pull request merged**